itenium-socks/src/app/socks/sock-reviews.component.ts

```diff
@@ -11,11 +11,16 @@ import { Review } from './sock.model';
   templateUrl: './sock-reviews.component.html'
 })
 export class SockReviewsComponent {
+  @Input() sockId!: number | undefined;
```

Comment:
Het `!` wil zeggen dat de value altijd gezet is (voor TypeScript, niet voor Angular)
Voor Angular is dat met `@Input({required: true}) sockId: number`

Aangezien je hier undefined ook accepteert

```
@Input() sockId: number;
```